### PR TITLE
Added title & challenge num to lesson cards and sorted using order

### PR DESCRIPTION
**Pull request by aortizoj15**

### CODE REVIEW

File: pages/curriculum.tsx [hunk 1]
```diff
@@ -13,15 +13,20 @@ const Curriculum = () => {
 
   if (data) {
     const { lessons } = data
+    console.log(lessons)
```
Comment (songz) [inline pages/curriculum.tsx]: remove `console.log`

File: pages/curriculum.tsx [hunk 2]
```diff
@@ -13,15 +13,20 @@ const Curriculum = () => {
...
     return (
       <Layout>
-        <Card title="Curriculum">
-          <ul>
-            {lessons.map((e: any) => (
-              <li key={e.id}>{e.title}</li>
-            ))}
-          </ul>
-        </Card>
+        {lessons
```
Comment (songz) [inline pages/curriculum.tsx]: please don't put logic inside the rendering component. Create a variable like `lessonList` before `return` and render variable here. This helps keep the component code clean.

File: pages/curriculum.tsx [hunk 3]
```diff
@@ -13,15 +13,20 @@ const Curriculum = () => {
...
+          .sort((a: any, b: any) => a.order - b.order)
```
Comment (hwong0305) [inline pages/curriculum.tsx]: Don't use `any` type when you know the type of the item. This defeats the purpose of having TypeScript. 

For this case you should be declaring the type of lessons like

`type Lesson = {
  order: number
   title: string
    ...
}`

File: components/Layout.tsx [hunk 4]
```diff
@@ -3,7 +3,7 @@ import AppNav from './AppNav'
 import Footer from './Footer'
 
 type Props = {
-  children: React.ReactElement
+  children: React.ReactElement | {}
```
Comment (hwong0305) [inline components/Layout.tsx]: Can you remove this? We don't want Layout to be blank. Layout should always have children.

File: pages/curriculum.tsx [hunk 5]
```diff
@@ -4,24 +4,38 @@ import Layout from '../components/Layout'
...
     return (
       <Layout>
-        <Card title="Curriculum">
-          <ul>
-            {lessons.map((e: any) => (
-              <li key={e.id}>{e.title}</li>
-            ))}
-          </ul>
-        </Card>
+        {sortedLessons.map((e: Lesson) => (
```
Comment (hwong0305) [inline pages/curriculum.tsx]: You could have some placeholder text for the other boxes.

File: pages/curriculum.tsx [hunk 6]
```diff
@@ -4,24 +4,38 @@ import Layout from '../components/Layout'
...
+type Lesson = {
+  id: number
+  title: string
+  description: string
+  order: number
+  challenges: Challenge[]
+}
 
-const Curriculum = () => {
+const Curriculum: React.FC<any> = () => {
```
Comment (Wolfy64) [inline pages/curriculum.tsx]: If you don't have props you can just do `const Curriculum: React.FC = () => {}`

File: pages/curriculum.tsx [hunk 7]
```diff
@@ -4,24 +4,42 @@ import Layout from '../components/Layout'
...
-        <Card title="Curriculum">
-          <ul>
-            {lessons.map((e: any) => (
-              <li key={e.id}>{e.title}</li>
-            ))}
-          </ul>
-        </Card>
+        <>
+          {sortedLessons}
+          <Card title="Progress"></Card>
```
Comment (hwong0305) [inline pages/curriculum.tsx]: Placeholder for #50

File: pages/curriculum.tsx [hunk 8]
```diff
@@ -4,24 +4,42 @@ import Layout from '../components/Layout'
...
+          <Card title="General Announcements"></Card>
```
Comment (hwong0305) [inline pages/curriculum.tsx]: Placeholder for #51

**Pull request merged**